apps/ledger-live-mobile/src/components/RootNavigator/EarnLiveAppNavigator.tsx

```diff
@@ -111,7 +111,7 @@ export default function EarnLiveAppNavigator(_props?: Record<string, unknown>) {
       <Stack.Screen
         name={ScreenName.Earn}
         options={{
-          headerStyle: styles.headerNoShadow,
+          headerShown: false,
```

Comment:
When the user navigates to another screen via e.g. the "stake" button, the bottom navigation is removed. Then when you press "close" for that drawer or screen, you are taken back to this screen with no way out: 
![image](https://github.com/LedgerHQ/ledger-live/assets/123105524/de221853-f645-4a3f-81c1-f1bc2e9bc4ca)

So we may need to conditionally enable the header when routes are present, or make sure that the lower navigation comes back on closing the other routes. Might need a separate ticket for that though, so need not prevent this going through behind the feature flag

Reply:
great spot, need to fix that